integration-tests/tests/hacbs/add-component-to-application-via-hacbs-tab.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Applications } from "../../utils/Applications";
+import { Common } from "../../utils/Common";
+import { ComponentsTabPage } from "../../support/pages/hacbs/ComponentsTabPage";
+import { actionsDropdown, onboardingModalPO } from "../../support/pageObjects/hacbs-po";
+import { HACBSApplications } from "../../utils/HACBSApplications";
+
+describe('Create Components using the HACBS UI', () => {
+    const applicationName = Common.generateAppName();
+    const componentsTabPage = new ComponentsTabPage();
+    const publicRepos = [
+            'https://github.com/dheerajodha/devfile-sample-code-with-quarkus',
+            'https://github.com/devfile-samples/devfile-sample-go-basic.git',
+            'https://github.com/nodeshift-starters/devfile-sample.git'
+            ]
+    const componentNames = ['java-quarkus', 'go', 'nodejs'];
+
+    before(function () {
+        // Enable HACBS
+        localStorage.setItem('hacbs', 'true');
+        // Need to reload the page after enabling HACBS via localStorage
+        cy.reload();
+
+        // Go through onboarding modals
+        cy.get(onboardingModalPO.footerButton).click();
+        cy.get(onboardingModalPO.footerButton).click();
+        cy.get(onboardingModalPO.footerButton).click();
```

Comment:
yes, I thought let's at least test the onboarding modals at least once. But now that I think again, it's not that important because we won't be having any significant breaking changes on the onboarding modals in the future so we can skip testing them. Thanks for the suggestion.